src/unitxt/processors.py

```diff
@@ -75,6 +75,15 @@ def process(self, instance):
         return splitted[0].strip()
 
 
+class ConvertToBoolean(BaseFieldOperator):
+    def process(self, instance):
```

Comment:
where is this new operator used? could you please point me to an example of it? is it in your other PR in FM-Eval?

Reply:
Also, please add tests for the new operator..

Reply:
the new operator is used in the BoolQ_robustness dataset card in prepare/cards/boolq_robustness.py in fm-eval

added a test for the new processor in tests/test_postprocessors.py

Reply:
> the new operator is used in the BoolQ_robustness dataset card in prepare/cards/boolq_robustness.py in fm-eval

[Here](https://github.ibm.com/IBM-Research-AI/fm-eval/blob/robustness-datasets-local/prepare/cards/boolq_robustness.py)?

Reply:
oh ok, found it!